### DISCUSSION TITLE
Wallet connect doesnt show scannable qr code

### DISCUSSION THREAD

**Discussion by deantek** [wevm/wagmi#2010] [Q&A]
I do everything according to the tutorial, metamask works fine, but wallet connect does not, help pls
![image](https://user-images.githubusercontent.com/37242034/225658407-ab226c15-e7a7-4b57-9a7c-13ca6494f080.png)
```
const { chains, provider, webSocketProvider } = configureChains(
  [mainnet],
  [infuraProvider({ apiKey: '...' }), publicProvider()],
)`

const client = createClient({
  autoConnect: true,
  connectors: [
    new MetaMaskConnector({ chains }),
    new WalletConnectConnector({
      chains: [mainnet],
      options: {
        projectId: '...',
      },
    }),
  ],
  provider,
  webSocketProvider,
})
```

**Comment by web3dopamine** [comment]
add
` qrcode: true,`

```
const { chains, provider, webSocketProvider } = configureChains(
  [mainnet],
  [infuraProvider({ apiKey: '...' }), publicProvider()],
)`


const client = createClient({
  autoConnect: true,
  connectors: [
    new MetaMaskConnector({ chains }),
    new WalletConnectConnector({
      chains: [mainnet],
      options: {
        qrcode: true,
        projectId: '...',
      },
    }),
  ],
  provider,
  webSocketProvider,
})
`

```

**Reply by web3dopamine** [reply]
also are you adding Project id???

**Reply by deantek** [reply]
> also are you adding Project id???

yes, i added projectId, also added qrcode: true, but ts says there is no such option, i did //@ts-ignore, nothing changed anyway

**Reply by web3dopamine** [reply]
> > also are you adding Project id???
> 
> yes, i added projectId, also added qrcode: true, but ts says there is no such option, i did //@ts-ignore, nothing changed anyway

oh yeah that's for legacy, my bad : https://wagmi.sh/react/connectors/walletConnectLegacy
